subjects/devops/cloud-design/README.md

```diff
@@ -1,120 +1,206 @@
+This section will involve your answering a series of questions in a simulated
+real-world scenario, where you will assume the role of a Cloud engineer
+explaining your solution to a team or a stakeholder.
 
-To enhance the learning experience and assess your knowledge, a roleplay question session will be included as part of the Cloud-Design Project. This section will involve answering a series of questions in a simulated real-world scenario where you assume the role of a Cloud engineer explaining your solution to a team or stakeholder.
+To enhance the learning experience and assess your knowledge, a role play
+question session will be included as part of the Cloud-Design Project. This
+section will involve answering a series of questions in a simulated real-world
+scenario where you assume the role of a Cloud engineer explaining your solution
+to a team or stakeholder.
 
-The goal of the roleplay question session is to:
+The goal of the role play question session is to:
 
-- Assess your understanding of the concepts and technologies used in the project.
+- Assess your understanding of the concepts and technologies used in the
+  project.
 - Test your ability to communicate effectively and explain your decisions.
-- Challenge you to think critically about your solution and consider alternative approaches.
+- Challenge you to think critically about your solution and consider
+  alternative approaches.
 
-Prepare for a roleplay question session where you will assume the role of a Cloud engineer presenting your solution to your team or a stakeholder. You should be ready to answer questions and provide explanations about your decisions, architecture, and implementation.
+Prepare for a role play question session where you will assume the role of a
+Cloud engineer presenting your solution to your team or a stakeholder. You
+should be ready to answer questions and provide explanations about your
+decisions, architecture, and implementation.
 
 ### Architecture
 
-By using your solutions in your previous projects `crud-master`, `play-with-containers`, and `orchestrator` you have to design and deploy the infrastructure on AWS respecting the project requirements, consisting of the following components:
-
-- `inventory-database container` is a PostgreSQL database server that contains your inventory database, it must be accessible via port `5432`.
-- `billing-database container` is a PostgreSQL database server that contains your billing database, it must be accessible via port `5432`.
-- `inventory-app container` is a Node.js server that contains your inventory-app code running and connected to the inventory database and accessible via port `8080`.
-- `billing-app container` is a Node.js server that contains your billing-app code running and connected to the billing database and consuming the messages from the RabbitMQ queue, and it can be accessed via port `8080`.
+By using your solutions in your previous projects `crud-master`,

```

Comment:
I have clarified the point in the bonus part by adding all the projects instead of only `orchestrator`. I guess that should be fine - let me know if there is anything else you think I should add